java-squid/src/main/java/org/sonar/java/se/ConstraintManager.java

```diff
@@ -73,7 +76,7 @@ public SymbolicValue eval(ProgramState programState, Tree syntaxNode) {
       }
       case BOOLEAN_LITERAL: {
         boolean value = Boolean.parseBoolean(((LiteralTree) syntaxNode).value());
-        if(value) {
+        if (value) {
           return SymbolicValue.TRUE_LITERAL;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 13. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)